pkg/kubelet/metrics/metrics.go

```diff
@@ -515,7 +515,9 @@ func SetActiveConfig(source *corev1.NodeConfigSource) error {
 		return err
 	}
 	// clean up the old timeseries (WithLabelValues creates a new one for each distinct label set)
-	ActiveConfig.Delete(activeConfigLabels)
+	if ret := ActiveConfig.Delete(activeConfigLabels); !ret {
+		klog.Warningf("unable to delete active config labels")
```

Comment:
Can you do the same for the other Set*Config helpers in this file?

Let's make the messages something like "Unable to delete {active,assigned,last known good} config metric for labels %s. Either the metric did not exist or an error prevented deletion."

Reply:
I also like David's suggestion about explaining what could break; in this case if multiple label sets are registered, it becomes ambiguous which config is in use.